plugins/rikki/loungeviews/components/PlayoffOverview.php

```diff
@@ -136,13 +141,19 @@ public function onRender()
         $this->timeFormat = TimezoneHelper::getTimeFormatString();
     }
 
-    //gets offsets in rem for a playoff_position $pp
-    public function getOffsetsForMatch($pp)
+    private function getBracketName($dec_position) {
```

Comment:
For consistency, lets put the bracket in its own line

Reply:
updated